apps/authentication-service/cypress/e2e/clients.cy.ts

```diff
@@ -0,0 +1,74 @@
+import { Client } from "auth0";
+
+describe("Client API", () => {
+  const PORT: number = (Cypress.env("PORT") as number) || 8089;
+  let accessToken: string;
+  let newClientId: string;
+
+  const getToken = () => {
+    cy.window()
+      .its("localStorage")
+      .invoke("getItem", "jwt")
+      .then((token) => {
+        if (token) {
+          accessToken = token;
+        }
+      });
+  };
+
+  before(() => {
+    getToken();
+    if (!accessToken) {
+      cy.loginByAuth0Api();
+      cy.wait(1500);
+      getToken();
+    }
+  });
+
+  it("creates new client", () => {
+    const clientName = "__Test Client__";
+
+    cy.request({
+      method: "POST",
+      url: `http://localhost:${PORT}/v1/clients`,
+      headers: {
+        ContentType: "application/json",
+        Authorization: `Bearer ${accessToken}`,
+      },
+      body: {
+        name: clientName,
+        description: "--DELETE ME--",
+      },
+    }).then((response: Cypress.Response<{ body: Client }>) => {
+      newClientId = (response.body as unknown as Client).client_id;
+      expect(response.status).to.eq(200);
+      expect(response.body).property("name").to.eq(clientName);
+    });
+  });
+
+  it("returns client info", () => {
+    cy.wait(1500);
+    cy.request({
+      method: "GET",
+      url: `http://localhost:${PORT}/v1/clients/${newClientId}`,
```

Comment:
I don't think these are the api signatures we're looking for. 

When we log in we don't know our own clientId, so we can't ping an api to get our secret.

The way this is built if someone knew another person's clientId, then they could get their secrets and delete their clients.

These 3 apis should really be createMySecret/Client, getMySecret/Client, and deleteMySecret/Client.